Command/command_factory.py

```diff
@@ -0,0 +1,29 @@
+import sys,os
+sys.path.append(os.path.join(os.getcwd(), "Functions"))
+from CreateCommand import *
+from SetCommand import *
+from GetCommand import *
+from DeleteCommand import *
```

Comment:
class name should be like what you did, however, file name should follow this convenstion in python 
`create_command.py`